server/storage/backend.go

```diff
@@ -35,13 +35,13 @@ func newBackend(cfg config.ServerConfig, hooks backend.Hooks) backend.Backend {
 	if cfg.BackendBatchLimit != 0 {
 		bcfg.BatchLimit = cfg.BackendBatchLimit
 		if cfg.Logger != nil {
-			cfg.Logger.Info("setting backend batch limit", zap.Int("batch limit", cfg.BackendBatchLimit))
+			cfg.Logger.Info("setting Backend batch limit", zap.Int("batch limit", cfg.BackendBatchLimit))
```

Comment:
Curious what is the reason for this uppercase? 🤔

Reply:
Intelij is dumb, it thinks that every mention of name must be related to function or struct. So when you rename "backend" -> "Backend" (eh golang), all your comments and strings get also updated. More silly is when you rename more common words "add", "get" etc.